modules/virt-creating-offline-vm-snapshot-web.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-managing-offline-vm-snapshots.adoc
+
+[id="virt-creating-offline-vm-snapshot-web_{context}"]
+= Creating an offline virtual machine snapshot in the web console
+
+You can create a virtual machine (VM) snapshot by using the web console.
+
+.Procedure
+
+. Click *Workloads* → *Virtualization* from the side menu.
+
+. Click the *Virtual Machines* tab.
+
+. Select a virtual machine to open the *Virtual Machine Overview* screen.
+
+. If the virtual machine is running, click *Actions* → *Stop Virtual Machine* to power it down.
+
+. Click the *Snapshots* tab, then click *Take Snapshot*.
+
+. Fill in the *Snapshot Name* and optional *Description* fields.
+
+. Expand *Disks included in this Snapshot* to see the storage volumes to be included in the snapshot.
++
+[NOTE]
+====
+The VM snapshot only includes disks that:
+
+* Are the type `dataVolume` or `persistentVolumeClaim` and
+* Belong to a storage class that supports Container Storage Interface (CSI) volume snapshots
+
+If your VM storage includes disks that do not support snapshots, you can either edit them or contact your cluster administrator.
+====
+
+. Select the *I am aware of this warning and wish to proceed* checkbox to proceed.
```

Comment:
This checkbox will only appear if there are disks that cannot be snapshotted, so maybe you should state its conditional.